src/twisted/web/test/test_distrib.py

```diff
@@ -395,36 +395,32 @@ def test_interface(self):
         """
         self.assertTrue(verifyObject(resource.IResource, self.directory))
 
-    def _404Test(self, name):
+    async def _404Test(self, name: bytes) -> None:
```

Comment:
I am not happy with this helper... but is already in trunk, so it should not be a blocker.

The `_404Test` name doesn't disclose too much what this call is all about.

I think that having it start with `check` or `assert` should make it easier to understand what is the purpose of this method.

And all this, without having to guess that `self.directory` is automatically used.


```suggestion
    async def check404Render(self, target, name: bytes) -> None:
```

Reply:
IMO this doesn't matter &mdash; this is not public API, it's used in two tests immediately below. Anyone interested in it can read the docstring (which mentions `self.directory`) or the entire 5-line implementation.